cns/restserver/restserver.go

```diff
@@ -1032,6 +1032,8 @@ func (service *HTTPRestService) saveNetworkContainerGoalState(req cns.CreateNetw
 	case cns.JobObject:
 		fallthrough
 	case cns.COW:
+		fallthrough
+	case cns.WebApps:
```

Comment:
- Is there a GET call to get the state?
- Did you check the same bug doesn't exist on delete?
- Could you add a test case?

Reply:
How is this helping, is the orchestrator type handling for webapps?

Reply:
Do you need to handle orchestrator type case?

Reply:
Does the state need to be saved? considering it is a sync operation?
Yes. Already answered in the email

Is there a GET call to get the state?
No. This is no different from what we have for other first party work flows

Did you check the same bug doesn't exist on delete?
It works.

Could you add a test case?
Do we have test case for other container types? I don't think so.

Do you need to handle orchestrator type case?
Good catch. Added it. I used k8s as orch on test cluster so it didn't show up.